src/main/java/com/minecolonies/coremod/compatibility/jei/transfer/PrivateCraftingTeachingTransferHandler.java

```diff
@@ -122,9 +120,9 @@ public IRecipeTransferError transferRecipe(
             return handlerHelper.createUserErrorWithTooltip(tooltipMessage);
         }
```

Comment:
I didn't change this existing logic but it seems very dubious to me.  It's basically preventing any teaching of recipes at all if the `doLimitedCrafting` game rule is enabled -- fortunately few people enable that, but still.

It appears this logic was introduced by 330967179; prior to that it was only returning the error if the recipe was not already in the player's recipe book, which makes more sense.

Reply:
I'd say remove it. Doesn't make sense to restrict this here

Reply:
Ok, done.